sdk/tests/conformance2/glsl3/short-circuiting-in-loop-condition.html

```diff
@@ -81,7 +81,7 @@
   if (!gl) {
     testFailed("context does not exist");
   } else {
-    var program = wtu.setupProgram(gl, ["vertex-shader", "fshader"]);
+    var program = wtu.setupProgram(gl, ["vertex-shader", "fshader"], ['aPosition'], undefined, true);
```

Comment:
toji@ can we add a general logShaders flag to webgl conformance suite, so we can turn it on for debugging and off when running on bots?